src/environment/EnvironmentManager.py

```diff
@@ -55,28 +55,32 @@ def setup(env_name: str) -> bool:
         return res == 0
 
     @staticmethod
-    def run(env_name: str, *args, **kwargs) -> int:
+    def run(agent_name: str, env_name: str = None, *args, **kwargs) -> int:
         """
-        This function will start the agent of the specified environment in a new system process.
-        For this to work, there must be a file `environment/agent_{env_name}.py` which will
-        be launched in the specified conda environment. Note that the environment must be
-        created through `EnvironmentManager.setup(env_name)` first.
+        This function will start the specified agent(named `agent_name`) of the specified
+        environment in a new system process.For this to work,
+        there must be a file `environment/agent_{agent_name}.py` which will be launched in
+        the specified conda environment. Note that the environment must be created through
+        `EnvironmentManager.setup(env_name)` first.
 
         All trailing arguments and keyworks arguments will be passed to the agent in a CLI format.
 
         Args:
+            agent_name (str): The name of the file to run.
             env_name (str): The name of the environment to run.
```

Comment:
maybe mention that `None` is a valid value, and what happens if this is the case